src/views/VallisFish.vue

```diff
@@ -2,171 +2,141 @@
   <b-container fluid>
     <b-col md="12">
       <router-link to="/vallis/map">
-        <b-button variant="info" class="btn-block mb-3">Orb Vallis Map</b-button>
+        <b-btn variant="info" class="btn-block mb-3">Orb Vallis Map</b-btn>
       </router-link>
 
       <div class="row">
-        <b-table striped hover :items="items" class="fish-info b-table"></b-table>
+        <table class="table fish-info striped hover">
+          <thead>
+            <tr>
+              <th title="The name of the fish">Fish Name</th>
+              <th title="Number of scrap you receive is dependent on fish model">
+                Model
+              </th>
+              <th title="Number of scrap you will receive when dismantling">
+                Scrap
+              </th>
+              <th title="You will always receive 1 of these per fish when dismantling">
+                Unique
+              </th>
+              <th title="Solaris United standing gain when donating fish">Standing</th>
+              <th title="Location of where to find the fish">Location</th>
+              <th title="Time of when you can find the fish">Time</th>
+              <th title="How likely the fish will spawn">Rarity</th>
+              <th title="The appropriate spear needed for this fish">Spear</th>
+              <th title="The maximum point possible for this fish">
+                Max Point
+              </th>
+            </tr>
+          </thead>
+          <tbody v-for="(fish, index) in this.fishes" :key="fish.name">
+            <tr :class="'color' + ((index % 2) + 1)">
```

Comment:
Can we use the `:even` and `:odd` pseudoclasses instead of dynamically generating the class names?

Reply:
Yeah, would prefer that

Reply:
I have gone ahead and omitted this one in the latest PR. This does not work as nicely in our case because we use rowspan.